### DISCUSSION TITLE
Key issue

### DISCUSSION THREAD

**Discussion by minima1cha0s** [community/community#63941] [New to GitHub]
Hi, I need help with keys.

I initially had a private repo for instance "ABC". 
I had added a deploy key to it.
I have deleted the repo.

Now I recreated a private repo with the same name "ABC". I realized the deploy keys are per repo so I am trying to add my previous key as a ssh key for authentication. I get 2 messages - Key already exists and Key successfully added. But upon refresh of the page no key is actually added.

Does it have anything to do with me not removing the previous deploy key before deleting the repo ? Deleting a repo with deploy keys does not delete the keys ?

**Comment by DylanDevelops** [comment]
Try going to your `Account settings > SSH and GPG keys` and see if any of your keys are in there. You may be able to delete them. let me know if this fixes it.
